integ/lib/render-struct.ts

```diff
@@ -79,11 +86,11 @@ export class RenderStruct extends Construct {
       vpc,
       repository: props.repository,
       images: recipes.renderQueueImages,
-      version: recipes.version,
       logGroupProps: {
         logGroupPrefix: Stack.of(this).stackName + '-' + id,
       },
       hostname,
+      version: props.version,
```

Comment:
I'm thinking that, for now, we should keep the RenderQueue's version as the version from the recipe for now. Reason being that we want the RenderQueue's version to match the staged version.

What do you think?

Reply:
Ah yes, I just made a similar reply to your previous comment